docs/testrunner-toolkit/configuration/espresso.md

```diff
@@ -1,69 +1,302 @@
+---
+
+### `download`
+<p><small>| OPTIONAL | OBJECT |</small></p>
+
+Specifies the settings related to downloading artifacts from tests run by `saucectl`.
+
+```yaml
+  download:
+    when: always
+    match:
+      - junit.xml
+    directory: ./artifacts/
+```
+---
+
+#### `when`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Specifies when and under what circumstances to download artifacts. Valid values are:
+
+* `always`: Always download artifacts.
+* `never`: Never download artifacts.
+* `pass`: Download artifacts for passing suites only.
+* `fail`: Download artifacts for failed suites only.
+
+```yaml
+    when: always
+```
+---
+
+#### `match`
+<p><small>| OPTIONAL | STRING/ARRAY |</small></p>
+
+Specifies which artifacts to download based on whether they match the name or file type pattern provided. Supports the wildcard character `*` so you can conveniently specify all artifacts of a specific file type.
+
+```yaml
+  match:
+    - junit.xml
+    - *.log
+```
+---
+
+#### `directory`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Specifies the path to the folder location in which to download artifacts. A separate subdirectory is generated in this location for each suite for which artifacts are downloaded.
+
+```yaml
+    directory: ./artifacts/
+```
+---
+
+## `espresso`
+<p><small>| REQUIRED | OBJECT |</small></p>
 
-__Type__: *object*
+The parent property containing the details specific to the Espresso project.
 
-__Example__:
 ```yaml
 espresso:
   app: ./apps/calc.apk
   testApp: ./apps/calc-success.apk
 ```
+---
 
 ### `app`
+<p><small>| REQUIRED | STRING |</small></p>
 
-__Description__: Path to the application. It supports expanded enviornment variable.
-
-__Type__: *string*
+The directory path to the application. The default directory is `{project-root}/apps/filename.apk`, and the property supports expanded environment variables to designate the path, as shown in the following examples.
 
-__Examples__:
 ```yaml
   app: ./apps/calc.apk
 ```
 
 ```yaml
   app: $APP
 ```
+---
 
 ### `testApp`
+<p><small>| REQUIRED | STRING |</small></p>
 
-__Description__: Path to the testing application. It supports expanded enviornment variable.
-
-__Type__: *string*
+The directory path to the testing application. The default directory is `{project-root}/apps/testfile.apk`, and the property supports expanded environment variables to designate the path, as shown in the following examples.
 
-__Examples__:
 ```yaml
   testApp: ./apps/calc-success.apk
 ```
 
 ```yaml
   testApp: $TEST_APP
 ```
+---
 
-## `emulators`
+## `suites`
+<p><small>| REQUIRED | OBJECT |</small></p>
 
-__Description__: Field for defining emulator details such as the name, orientation, and
-platformVersions.
+The set of properties providing details about the test suites to run. May contain multiple suite definitions. See the full [example config](#example-configuration) for an illustration of multiple suite definitions.
 
-__Type__: *object*
+:::tip Configure RDC and VMC
```

Comment:
This is another one I have been confused about. I remember reading in a few Slack channels (maybe the UI Design channel) that we wanted to move away from "virtual device" because it is not really a device, so we were supposed to adopt the terms "real device cloud" and "virtual machine cloud" -- but I am not seeing that adoption in very many places, so maybe I am wrong? We should take this issue to a bigger audience, maybe.

Reply:
@KunalJain Any opinion on that ?